### PR TITLE
AWS Summit Singapore April 2018

### PR DESCRIPTION
**Pull request by gayathri-krishnaswamy**
Hello,

I am new to GITHUB and I found your project very interesting. I'd like to actively contribute to your codeandtalk project. I am a technical writer by profession. I created a new event for AWS Summit to be held in Singapore on April 4 2018.

{
    "accessibility": "",
    "cfp_end": "",
    "code_of_conduct": "",
    "comment": "",
    "diversitytickets": "",
    "diversitytickets_text": "",
    "diversitytickets_url": "",
    "event_end": "2018-04-04",
    "event_start": "2018-04-04",
    "facebook": "",
    "hashtag": "",
    "languages": [
        "English"
    ],
    "location": {
        "city": "Singapore",
        "country": "Singapore",
        "state": "Singapore"
    },
    "name": "AWS Summit",
    "tags": [],
    "twitter": "AWSCloudSEAsia",
    "website": "https://aws.amazon.com/summits/singapore/",
    "youtube": ""
}
© 2018 GitHub, Inc.

### CODE REVIEW

File: data/events/aws_summit_singapore_april4_2018 [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        "country": "Singapore",
+        "state": "Singapore"
+    },
+    "name": "AWS Summit",
+    "tags": [],
+    "twitter": "AWSCloudSEAsia",
+    "website": "https://aws.amazon.com/summits/singapore/",
+    "youtube": ""
+}
+© 2018 GitHub, Inc.
```
Comment (rollandf) [inline data/events/aws_summit_singapore_april4_2018]: Remove this line

File: data/events/aws_summit_singapore_april4_2018 [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (rollandf) [inline data/events/aws_summit_singapore_april4_2018]: The file name should be like this:
aws-summit-singapore-2018.json

**Comment by rollandf** [comment]
the file should have a suffix of .json

**Comment by szabgab** [comment]
Thanks for updating the filename and for your submission.

I have no idea why Coveralls found that our test coverage has decreased. Especially given that there is no code or test change. I'll have to investigate that later.

Your contribution can be seen here: https://codeandtalk.com/e/aws-summit-singapore-2018

Thank you!

**Pull request merged**